### PR TITLE
Mobile: Implemented media options sheet

### PR DESCRIPTION
**Pull request by marecar3**
Introduced media options sheet based on Bottom sheet component.

Test : Gutenberg mobile PR : https://github.com/wordpress-mobile/gutenberg-mobile/pull/539

ezgif com-video-to-gif 3

### CODE REVIEW

**Comment by SergioEstevao** [comment]
Why do we only show the `upload photo` option when we add the image block?

File: packages/editor/src/components/media-placeholder/index.native.js [hunk 1]
```diff
@@ -1,27 +1,26 @@
...
-				<Button title={ __( 'Device Library' ) } onPress={ props.onUploadMediaPressed } />
-				<Button title={ __( 'Take photo' ) } onPress={ props.onCapturePhotoPressed } />
+		<TouchableWithoutFeedback onPress={ props.onMediaOptionsPressed }>
+			<View style={ styles.emptyStateContainer }>
+				<Dashicon icon={ 'cover-image' } />
+				<Text style={ styles.emptyStateTitle }>
+					{ __( 'Image' ) }
+				</Text>
+				<Text style={ styles.emptyStateDescription }>
+					{ __( 'UPLOAD IMAGE' ) }
```
Comment (SergioEstevao) [inline packages/editor/src/components/media-placeholder/index.native.js]: Maybe change this string to: "Select Image" ?

File: packages/block-library/src/image/edit.native.js [hunk 2]
```diff
@@ -214,6 +216,47 @@ export default class ImageEdit extends React.Component {
...
+					onPress={ onMediaCaptureButtonPressed }
+				/>
+				<BottomSheet.Cell
+					label={ __( 'WordPress Media Library' ) }
+					labelStyle={ { color: '#00aadc' } }
+					onPress={ onMediaLibraryButtonPressed }
+				/>
+				<BottomSheet.Cell
+					label={ __( 'Dismiss' ) }
+					labelStyle={ { color: '#00aadc ', fontWeight: 'bold' } }
```
Comment (etoledom) [inline packages/block-library/src/image/edit.native.js]: There's an extra space in the color that generates a warning:

screen shot 2019-02-05 at 9 02 20 am

screen shot 2019-02-05 at 8 58 41 am

```suggestion
					labelStyle={ { color: '#00aadc', fontWeight: 'bold' } }
```

**Review by etoledom — COMMENTED** [review]
Thank you for this PR @marecar3 !

I agree with @SergioEstevao that the button title should not be `Upload` since we are not always uploading images (i.e. WP Media Library).

Do we need extra work to apply these changes to WPiOS? The buttons are not working there.

I left one code comment with a small issue, but the rest looks good. 👍

File: packages/editor/src/components/media-placeholder/index.native.js (hunk 1 above)
Comment (marecar3) [reply packages/editor/src/components/media-placeholder/index.native.js]: Hey @SergioEstevao , thanks for the comment.
You are right. I will fix that.

File: packages/block-library/src/image/edit.native.js (hunk 2 above)
Comment (marecar3) [reply packages/block-library/src/image/edit.native.js]: @etoledom thanks for the comment.
@SergioEstevao is working on WPiOS and it will be ready soon.

**Comment by SergioEstevao** [comment]
Working great now on iOS!

**Review by SergioEstevao — APPROVED** [review]
Working great now on iOS!

**Pull request merged**